scope

```diff
@@ -81,7 +81,7 @@ check_docker_access() {
     fi
 }
 
-# - The image embeds the weave script & Docker 1.10.3 client
+# - The image embeds the weave script & Docker 1.13.1 client (mimicking a 1.10 client)
```

Comment:
If you wanted to, you could now reduce this number (i.e. support older versions), after suitable testing.